dipy/reconst/shm.py

```diff
@@ -965,3 +965,80 @@ def sh_to_sf_matrix(sphere, sh_order, basis_type=None, return_inv=True, smooth=0
         return B.T, invB.T
 
     return B.T
+
+def sh_to_ap(coeffs_mtx, normal_factor=0.00001):
+    """ Calculates anisotropic power map with a given SH coefficient matrix
+
+    Notes
+    ----------
+    Calculate AP image based on a IxJxKxC SH coeffecient matrix based on the equation:
+    .. math::
+        AP = \sum_{l=2,4,6,...}{\frac{1}{2l+1} \sum_{m=-l}{|a_{l,m}|^2}}
+
+    Where dim C is made of a flattened lxm coeffecient, where l are the SH levels.
+    and m = 2l+1.
+    So l=1 has 1 coeffecient, l=2 has 5, ... l=8 has 17 and so on.
+    A l=2 SH coeffecient matrix will then be composed of a IxJxKx6 volume. 
+
+    The final AP image is then normalized by log(AP/normal_factor). 
+    All values < 0 are discarded. 
+
+
+
+    References
+    ----------
+    .. [1]  Dell’Acqua, F., Lacerda, L., Catani, M., Simmons, A., 2014. 
+            Anisotropic Power Maps: A diffusion contrast to reveal low anisotropy tissues from HARDI data, 
+            in: Proceedings of International Society for Magnetic Resonance in Medicine. Milan, Italy.
+
```

Comment:
Yep - I believe that's the apostrophe in Flavio's name. Dell =>`<= Acqua. 

This is probably because the author list was copied over from non-ascii document (e.g. a pdf with the conference proceedings...). 

Please delete that one and put a standard ascii apostrophe.